packages/webcert/src/store/list/listMiddleware.test.ts

```diff
@@ -0,0 +1,102 @@
+import { configureStore, EnhancedStore } from '@reduxjs/toolkit'
+import reducer from '../reducers'
+import dispatchHelperMiddleware, { clearDispatchedActions } from '../test/dispatchHelperMiddleware'
+import MockAdapter from 'axios-mock-adapter'
+import axios from 'axios'
+import apiMiddleware from '../api/apiMiddleware'
+import { listMiddleware } from './listMiddleware'
+import { getDraftListConfig, getDrafts, ListResponse, performListSearch, updateActiveListFilter, updateActiveListType } from './listActions'
+import { CertificateListItem, ListType } from '@frontend/common/src/types/list'
+import { getConfigWithTextFilter, getDefaultList, getFilter, getFilterWithValues } from '../../feature/list/test/listTestUtils'
+
+const flushPromises = () => new Promise((resolve) => setTimeout(resolve))
+
+describe('Test list middleware', () => {
+  let fakeAxios: MockAdapter
+  let testStore: EnhancedStore
+
+  beforeEach(() => {
+    fakeAxios = new MockAdapter(axios)
+    testStore = configureStore({
+      reducer,
+      middleware: (getDefaultMiddleware) => getDefaultMiddleware().prepend(dispatchHelperMiddleware, apiMiddleware, listMiddleware),
+    })
+  })
+
+  afterEach(() => {
+    clearDispatchedActions()
+  })
+
+  describe('Handle get drafts', () => {
```

Comment:
Ensure tests aren't flaky by running them multiple times together with full suite. I don't trust these tests using `MockAdapter` and `fakeAxios.history` any more